mycore-mods/src/main/java/org/mycore/mods/MCRMODSWrapper.java

```diff
@@ -110,11 +117,14 @@ public static MCRObject wrapMODSDocument(Element modsDefinition, String projectI
      * @return true, if mods is supported
      */
     public static boolean isSupported(MCRObject obj) {
+        boolean supported;
```

Comment:
What's the advantage of this additional variable?
We could even further simplify and remove the if by
`return  isSupported(obj.getId()) || ...`